pydantic/mypy.py

```diff
@@ -761,6 +761,10 @@ def collect_field_or_class_var_from_stmt(  # noqa C901
                 )
                 node.type = AnyType(TypeOfAny.from_error)
 
+        if node.is_final and has_default:
+            # TODO this path should be removed (see https://github.com/pydantic/pydantic/issues/11119)
```

Comment:
Can you add a note about when, or what is blocking this?

Reply:
The linked issue should be enough?